django_redshift_backend/base.py

```diff
@@ -164,11 +184,27 @@ def create_model(self, model):
                     self.deferred_sql.extend(autoinc_sql)
 
         # Add any unique_togethers
-        for fields in model._meta.unique_together:
-            columns = [model._meta.get_field(field).column for field in fields]
-            column_sqls.append(self.sql_create_table_unique % {
-                "columns": ", ".join(self.quote_name(column) for column in columns),
-            })
+        if hasattr(self, 'sql_create_table_unique'):
```

Comment:
Dropped `sql_create_table_unique` since django-1.9 is not a part of supporting `UUIDField`. 
I fixed it on #18.